packages/frontend/electron/src/main/emitter.ts

```diff
@@ -0,0 +1,10 @@
+import type { TypedEventEmitter } from '@toeverything/infra/core/event-emitter';
```

Comment:
https://github.com/toeverything/AFFiNE/pull/5216/files#diff-5ee44df198617c4ac922685f1e6786e26713d887e7f1b03ba55a928bd7051d1dR7-R35

can you please check my changes in this closed PR? it no  longer uses event emitter and use dynamic import to get rid of circular dependency issue

Reply:
Got it